### ISSUE TITLE
Disable `<-`

### ISSUE THREAD

**Issue by goodmind** [tonsky/FiraCode#787]
This just looks weird in Flow
![image](https://user-images.githubusercontent.com/3275424/60775597-f828a100-a13d-11e9-8506-ad4846813d81.png)

**Comment by tonsky** [comment]
Okay thanks!

**Comment by unclechu** [comment]
But it looks as expected in Haskell. Would a fix for this remove `<-` completely?

**Comment by tonsky** [comment]
Yes, it’s one or the other. I have not decided on removing it yet, but I might. It mainly depends on relative popularity of Flow and Haskell

**Comment by goodmind** [comment]
I would prefer different ligatures for different languages

**Comment by unclechu** [comment]
@tonsky Maybe the right thing would be to add separated builds for specific programming languages?
If `<-` were removed it stops make sense to use it with Haskell anymore.

**Comment by tonsky** [comment]
It is not possible right now, but I hope to have separate build for languages in the future

**Comment by steebchen** [comment]
`<-` should not be disabled in my opinion, it looks great when using channels in Go:

![Selection_374](https://user-images.githubusercontent.com/5013932/61094653-5b714680-a450-11e9-8020-349990bfd6f3.jpg)

**Comment by Kharacternyk** [comment]
<- is the assignment operator in F#. It's a major usage case, IMHO.

**Comment by tonsky** [comment]
With Haskell, Go and F# I think Flow is overruled. I’ll keep it in the font until I have a way to do per-language builds. Thanks everyone!

**Comment by RubenVerg** [comment]
@tonsky Just asking, based on what can't u?

**Comment by RubenVerg** [comment]
If there's one branch, there can be infinite (well kinda).
What editor do you use for fonts?

**Comment by tonsky** [comment]
> Just asking, based on what can't u?

I don’t undestand the question

> What editor do you use for fonts?

Glyphs

**Comment by RubenVerg** [comment]
> I don’t undestand the question
If there is like 1 branch, you can create another with different glyphs, I think, right?

> Glyphs
Is it free?

**Comment by akiirui** [comment]
Easy way, turn off ligatures at Editor.

**Comment by goodmind** [comment]
@akiirui is it per-glyph or turns off all of them

**Comment by akiirui** [comment]
> @akiirui is it per-glyph or turns off all of them

Emmmm, sorry bro. is all of them.
But wanna disable `<-` for now, turn off ligatures.

In VSC:
```
    "editor.fontLigatures": true,
    "[flow]": {
        "editor.fontLigatures": false
    }
```

It's not perfect answer of course.
Waiting for a way to do per-language builds. 😉

**Comment by arknjl** [comment]
> With Haskell, Go and F# I think Flow is overruled. I’ll keep it in the font until I have a way to do per-language builds. Thanks everyone!

so far, the uses for `<-` where the ligature is favorable seems to be situations where it’s surrounded by white-space. possibly do the thing you did with disjunction and conjunction ligatures

**Comment by tonsky** [comment]
> possibly do the thing you did with disjunction and conjunction ligatures

It didn’t really worked. Editors split text by tokens and spaces, and inside that region there’s no way to tell what came before: letter/symbol or whitespace.

**Comment by euf** [comment]
It's worth noting that `<-` is the assignment operator in R, so it is used like in every expression. Please don't remove this ligature by default ❤️

**Comment by EPonyA** [comment]
GUI Emacs makes it simple to use different combinations of ligatures for different programming languages.
